flutter_todo/lib/main.dart

```diff
@@ -1,34 +1,26 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart' show rootBundle;
+import 'package:flutter_todo/realm/realm_services.dart';
 import 'dart:convert';
 import 'package:provider/provider.dart';
-import 'package:realm/realm.dart';
 import 'package:flutter_todo/realm/app_services.dart';
-import 'package:flutter_todo/realm/init_realm.dart';
 import 'package:flutter_todo/screens/homepage.dart';
 import 'package:flutter_todo/screens/log_in.dart';
 
 void main() async {
-  // get app id from config
   WidgetsFlutterBinding.ensureInitialized();
-  final realmConfig =
-      json.decode(await rootBundle.loadString('assets/config/realm.json'));
+  final realmConfig = json.decode(await rootBundle.loadString('assets/config/realm.json'));
   String appId = realmConfig['appId'];
   Uri baseUrl = Uri.parse(realmConfig['baseUrl']);
+ 
 
   return runApp(MultiProvider(providers: [
-    ChangeNotifierProvider<AppServices>(
-        create: (_) => AppServices(appId, baseUrl)),
-    ProxyProvider<AppServices, Realm?>(
-      update: (context, app, previousRealm) {
-        if (app.currentUser != null) {
-          previousRealm?.close();
-          return initRealm(app.currentUser!);
-        }
-        return null;
-      },
-      dispose: (_, realm) => realm?.close(),
-    )
+    ChangeNotifierProvider<AppServices>(create: (_) => AppServices(appId, baseUrl)),
+    ChangeNotifierProxyProvider<AppServices, RealmServices?>(
+        create: (context) => null, //RealmServices could be initialize only if the user is logged in.
```

Comment:
```suggestion
        create: (context) => null, // RealmServices could be initialize only if the user is logged in.
```

nit

Reply:
I will apply this in the new branch `desistefanova:flutter_app_v2`